server/filters/get-principle-main-media.js

```diff
@@ -1,10 +1,7 @@
-export default function getPrincipleMainMedia(arr) {
-  const video = arr.find((item) => {
-    return item.type === 'video';
-  });
-  const picture = arr.find((item) => {
-    return item.type === 'picture';
-  });
+export default function getPrincipleMainMedia(arr: []) {
+  const video = arr.find(item => item.type === 'video');
+  const mainPicture = arr.find(item => item.isMain);
```

Comment:
…yep, confirmed – `article.mainMedia` will be an array with two `Picture` items in it (because the 'featured image' ordinarily is the hero image). So I think this distinction is necessary.